docs/make.jl

```diff
@@ -122,6 +123,7 @@ makedocs(
         "Troubleshooting and FAQ" => "troubleshooting.md",
         "Reference" => [
                         "Trixi.jl" => "reference-trixi.md",
+                        "TrixiBase.jl" => "reference-trixibase.md",
```

Comment:
I cannot comment in the right line, but do we need `modules = [Trixi, Trixi2Vtk, TrixiBase]` in line 80?

Reply:
Idk, CI works without that as well. What's that line doing, anyway?

Reply:
The `modules` keyword [tests if all generated content has a docstring and prints a warning otherwise](https://documenter.juliadocs.org/stable/lib/public/#Documenter.makedocs). So with `TrixiBase` in the `modules` we ensure that TrixiBase.jl is covered by docstrings. I think it also specifies for which modules the doctests should be run. Without putting `TrixiBase` to `modules` the doctests from TrixiBase.jl do not run (and thus don't fail as we see in CI, cf. https://github.com/trixi-framework/TrixiBase.jl/issues/12). By putting it to `modules` the doctests run (and fail). I think if we put `Trixi2Vtk` inside `modules` we should also put `TrixiBase` (and fix the doctest there).